usr.bin/procstat/procstat_basic.c

```diff
@@ -46,7 +46,7 @@ procstat_basic(struct procstat *procstat __unused, struct kinfo_proc *kipp)
 {
 
 	if ((procstat_opts & PS_OPT_NOHEADER) == 0)
-		xo_emit("{T:/%5s %5s %5s %5s %5s %3s %-8s %-9s %-13s %-12s}\n",
```

Comment:
Hm, looks to me like there are only 79 characters used here, strangely, might be able to get away with leaving COMM as it is...

Reply:
Yes, this is true. Patch updated to take this into account.